src/crypto/BUILD.gn

```diff
@@ -60,7 +60,11 @@ static_library("crypto") {
   public_configs = []
   if (chip_crypto == "mbedtls") {
     sources += [ "CHIPCryptoPALmbedTLS.cpp" ]
-    public_deps += [ "${mbedtls_root}:mbedtls" ]
+
+    # Zephyr has its own integration of mbedTLS
+    if (current_os != "zephyr") {
```

Comment:
Can you move this condition to the mbedtls build file? Would like to avoid every target that depends on mbedtls to have to know where it comes from.

Reply:
Done.